packages/framework/src/Model/Customer/User/CustomerUserRefreshTokenChainRepository.php

```diff
@@ -38,4 +38,17 @@ public function getCustomersTokenChains(CustomerUser $customerUser): array
     {
         return $this->getCustomerUserRefreshTokenChainRepository()->findBy(['customerUser' => $customerUser]);
     }
+
+    /**
+     * @param string $deviceId
+     */
+    public function removeCustomerUserRefreshTokenChainsByDeviceId(string $deviceId): void
+    {
+        $this->em->createQueryBuilder()
+            ->delete(CustomerUserRefreshTokenChain::class, 'curtch')
```

Comment:
it should be curtc right? there is no "ch" letter in English :)

Reply:
Ok :smile: 4d1e65c044205c59a02704ce6a20c62f54952dff